gluster/cli/peer.py

```diff
@@ -47,6 +47,32 @@ def detach(host):
     cmd = ["detach", host]
     return peer_execute(cmd)
 
+def detach_all():
+    """
+    Removes All Hosts from Cluster
+
+    :returns: nothing
+     print: success and failure messages, raises
+    """
+    peers = parse_peer_status(peer_execute_xml(["status"]))
+    errors_list = []
+    if len(peers) > 0:
+        print "Found "+str(len(peers))+" peers"
+        for peer in peers:
+            host = peer["hostname"]
+            if peer["connected"] == "Connected":
+                cmd = ["detach",host]
+                try:
+                    result = peer_execute(cmd)
+                    out = str(host)+" "+result
+                    print out
```

Comment:
same as above comment. Output can be added to outlist and return this output as 

    return "\n".join(outlist)